src/org/thoughtcrime/securesms/jobs/SendJob.java

```diff
@@ -59,15 +63,19 @@ protected void markAttachmentsUploaded(long messageId, @NonNull List<Attachment>
 
     for (Attachment attachment : attachments) {
       try {
-        if (constraints.isSatisfied(context, masterSecret, attachment)) {
+        if (constraints.satisfiesCompression(context, attachment, TextSecurePreferences.getCompressionOptions(context))) {
+          MediaStream compressedFile = constraints.compressFile(context, masterSecret, attachment);
+          results.add(attachmentDatabase.updateAttachmentData(masterSecret, attachment, compressedFile));
+        }
+          else if (constraints.isSatisfied(context, masterSecret, attachment)) {
```

Comment:
`else if ` should be on the same line as the closing `}` of the `if statement`. Check the formatting below